modules/eap-xp-10-env/1.0.0/module.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "eap-xp-10-env"
+description: "JBoss Enterprise Application Platform XP 1.0 env and labels"
+version: "1.0.0"
+
+labels:
+    - name: "org.jboss.product"
+      value: "eap-xp"
+    - name: "org.jboss.product.version"
+      value: "1.0"
```

Comment:
regarding the value of the XP version, I think we should keep it with a x.y.z format similar to the EAP version(so it should be `1.0.0.`).

@kabir Could you confirm the XP version format please?

Reply:
Resolved in https://github.com/jboss-container-images/jboss-eap-7-image/pull/185